public/netsuke/assets/css/himotoshi.css

```
@@ -763,3 +931,162 @@ body {
+.hm-example-code-block .hm-syntax pre,
+.hm-example-terminal__body .hm-syntax pre {
+  padding: 0;
+}
+
+.hm-syntax { color: var(--netsuke-syntax-text); }
+.hm-syntax .c { color: var(--netsuke-syntax-comment); font-style: italic; }
+.hm-syntax .cp { color: var(--netsuke-syntax-comment-preproc); }
+.hm-syntax .k { color: var(--netsuke-syntax-keyword); }
+.hm-syntax .o { color: var(--netsuke-syntax-operator); }
+.hm-syntax .p { color: var(--netsuke-syntax-punctuation); }
+.hm-syntax .p p-Indicator { color: var(--netsuke-syntax-punctuation-indicator); }
```

Stylelint (17.14.1)

[error] Expected "p-Indicator" to be "p-indicator" (selector-type-case)

(selector-type-case)